netket/models/ndm.py

```diff
@@ -39,14 +39,24 @@ class PureRBM(nn.Module):
     """
 
     dtype: Any = np.float64
+    """The dtype of the weights."""
     activation: Any = nknn.logcosh
+    """The nonlinear activation function."""
     alpha: Union[float, int] = 1
+    """feature density. Number of features equal to alpha * input.shape[-1]"""
     use_hidden_bias: bool = True
+    """if True uses a bias in the dense layer (hidden layer bias)."""
     use_visible_bias: bool = True
+    """if True adds a bias to the input not passed through the nonlinear layer."""
+    precision: Any = None
+    """numerical precision of the computation see `jax.lax.Precision`for details."""
 
     kernel_init: Callable[[PRNGKey, Shape, Dtype], Array] = default_kernel_init
-    bias_init: Callable[[PRNGKey, Shape, Dtype], Array] = zeros
+    """Initializer for the Dense layer matrix."""
+    hidden_bias_init: Callable[[PRNGKey, Shape, Dtype], Array] = zeros
```

Comment:
Since you're modifying these lines: These could all be `netket.utils.typing.NNInitFunc`s, right?